### ISSUE TITLE
Tags reference to separate file not working 

### ISSUE THREAD

**Issue by nilkanth987** [asyncapi/spec#963]
Hello Guys, 

Referencing tags for operation object such as subscribe and publish is not working. Here is the sample code

```
# asyncapi.yml
asyncapi: 2.6.0
info:
  title: test API
  version: 0.0.21
servers:
  production:
    url: test.com/{basePath}
    description: Production server
    protocol: https
    security:
      - JWT: []
    variables:
      basePath:
        default: api/v2
defaultContentType: application/json
channels:
  /config:
    bindings:
      http:
        type: request
        method: GET
    subscribe:
      tags:
        - $ref: "test.yml#/tags/config"
        # ABOVE LINE IS NOT WORKING
      message:
        $ref: "#/components/messages/appConfigResp"
components:
  securitySchemes:
    JWT:
      type: http
      scheme: bearer
      bearerFormat: JWT
  messages:
    appConfigResp:
      name: appConfigResp
      payload:
        type: object
        properties:
          message:
            type: string
            description: Response message
          responseCode:
            type: number
            description: Response code
          success:
            type: boolean
            description: Is the response success
```

Below is the code of `test.yml`

```
# test.yml
tags:
  config:
    name: App Config
    description: App configs to control apps behaviour like force update
```

Below is the code for `app.js`

```
const generator = new Generator('@asyncapi/html-template', path.resolve('public/docs/ui'), { forceWrite: true, install: true })
generator.generateFromFile('asyncapi.yml')
```

I have tested the preview using `Asyncapi preview` extension in VSCode and it gives me proper result. Here is the screenshot of that

<img width="1229" alt="Screenshot 2023-08-11 at 3 21 52 PM" src="https://github.com/asyncapi/spec/assets/12413581/695dd6fb-b6ee-4aec-8f5c-d9b158d30f6b">

**_Let me know if I could help to fix this. Thanks._**

**Comment by nilkanth987** [comment]
Thanks for the update. Should I close this issue then?

**Comment by nilkanth987** [comment]
@magicmatatjahu , Thanks for the update. Should I close this issue then?

**Comment by magicmatatjahu** [comment]
@nilkanth987 Please :)

**Comment by nilkanth987** [comment]
References for tags will be available in 3.0.0
